COLLABORATOR_GUIDE.md

```diff
@@ -79,6 +79,7 @@ The Website also uses several other Open Source libraries (not limited to) liste
     - [PostCSS Simple Vars](https://github.com/postcss/postcss-simple-vars)
 - [Tailwind][] is used as our CSS Framework and the Foundation of our Design System
 - [Hero Icons](https://heroicons.com/) is an SVG Icon Library used within our Codebase
+- [Radix UI](https://www.radix-ui.com/) is a collection of customizable UI components
```

Comment:
```suggestion
- [Radix UI][] is a collection of customizable UI components
```
If the URL is being added at https://github.com/nodejs/nodejs.org/pull/5964/files#diff-ea0ecc2df076bab3682fd852974e605206ffbffd6fa35eb05e8e7db428efcf69R501 we should be able to just reference it here